internal/verifier/check.go

```diff
@@ -322,7 +322,7 @@ func (verifier *Verifier) CheckDriver(ctx context.Context, filter map[string]any
 			verifier.logger.Debug().
 				Msg("Received test's signal. Continuing.")
 		}
-		time.Sleep(verifier.generationPauseDelayMillis * time.Millisecond)
+		time.Sleep(verifier.generationPauseDelay * time.Millisecond)
```

Comment:
This and the `workerSleepDelay` were the only things I looked askance at here. Before, the name had `Millis` in it so you could sort of tell that you needed to convert it to milliseconds; now you can't tell that, so it feels very easy to be off accidentally  by a factor of a million.

It feels like `v.SetGenerationPauseDelayMillis` and `v.SetWorkerSleepDelayMillis` should just take an int64 directly (taken from the CLI context) and then set these fields to `time.Duration(arg) * time.Millisecond`, so that the stored durations are always correct.

Reply:
I changed the verifier method to `v.SetGenerationPauseDelay` and had it do the Millisecond multiplication there. That way (hopefully) it’ll be clearer what’s up.

Care to double-check the last commit?